Design3/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Testimonial Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="reviewers">
+        <div class="star">
+          <img src="dccl--responsive-testimonial-page/star_fill.svg" />
+        </div>
+        <div class="rContent">
```

Comment:
what is rContent

Reply:
review content

Reply:
make it review-content instead of rContent